Beheerlandschap.md

```diff
@@ -23,36 +25,17 @@ De relatie tussen deze omgevingen is gevisualiseerd in de onderstaande image:
 In GitHub bestaat voor elke API een repository maar ook voor enkele andere componenten. Zo hebben we nog een repository voor 'Zaken volgens GEMMA 2.0' dat dient als portaal naar alle gerelateerde GitHub repositories.
```

Comment:
Ik weet niet of voor elke API een repository bestaat. Bij convenant gemeenten hebben we veel kleine API's die niet allemaal een repository hebben. Misschien moet dat wel. Voorbeeld van zo'n kleine API is bij voorbeeld de check of iemand ouder is dan 18 jaar.

Reply:
Tja, beschrijven we hier de huidige werkelijkheid of de gewenste situatie? :)

Wij hanteren nu eigenlijk voor elk GEMMA component (zaken, documenten, besluiten, etc.) een repository plus een overkoepelende repository voor het concept (zaakgericht werken).

Typisch zit op zo'n component 1 API, dus effectief 1 API per repository.

Of dit uiteindelijk werkbaar is voor alle API's en componenten weet ik niet. Kan me ook wel voorstellen dat een component 2 API's heeft.

> Voorbeeld van zo'n kleine API is bij voorbeeld de check of iemand ouder is dan 18 jaar.

Is dat een volledige API? Klinkt als een functie bij een BRP?
Of stuur je een geboortedatum op naar `api/v1/18ofouder/` en krijg ja/nee terug?

Reply:
Het verschil is denk ik dat in de ZGW API's resources ontsloten worden in vrij low level systeem API's en de controle of iemand 18 jaar of ouder is een functionele vraag is die meer in een experience API laag thuis hoort. Omdat dit verschillende API's zijn die door eenzelfde systeem  geboden kunnen worden maar wel een eigen levenscyclus hebben zou ik ze in een aparte repository onderbrengen.  De experience API kan bijvoorbeeld doorontwikkelen terwijl de systeem API's niet hoeven te veranderen wanneer er in de resources niets veranderd.

Het hangt er ook van af of hetzelfde component de API's aanbiedt of dat de API's door verschillende componenten aangeboden worden. Door elke API specificatie in een eigen repository onder te brengen is het makkelijker de implementatie eventueel door aparte componenten te laten doen. Denk ik tenminste.

Reply:
Ik heb de tekst enigszins aangepast.
Kan ik de PR nu goedkeuren?